pkg/dockerutil/docker_manager.go

```diff
@@ -161,6 +161,15 @@ func ResetDockerHost(host string) error {
 	return nil
 }
 
+// GetServerVersion gets the cached info of Docker provider engine
```

Comment:
I assume this is copy-pasta. I guess it's getting the cached info of the semver version of the docker engine?

Reply:
This is the combined data that contains all versions from:
```
docker info
```